website/docs/cloud-docs/integrations/kubernetes/index.mdx

```diff
@@ -1,68 +1,168 @@
 
-## Prerequisites
+## Key benefits
 
-All Terraform Cloud users can use the Terraform Cloud Operator for Kubernetes. Some features of Terraform Cloud that are [limited to certain tiers](/terraform/cloud-docs/overview) aren't available to the Terraform Cloud Operator for Kubernetes unless you've purchased the corresponding tier.
+The Terraform Cloud Operator v2 offers several improvements over v1:
 
-## Terraform Cloud Operator for Kubernetes
+- **Flexible resource management**: The operator now features multiple custom resources, each with separate controllers for different Terraform Cloud resources. This provides additional flexibility and the ability to manage more custom resources concurrently, significantly improving performance for large-scale deployments.
 
-### Networking Requirements
+- **Namespace management**: The `--namespace` option allows you to tailor the operator's watch scope to specific namespaces, which enables more fine-grained resource management.
 
-In order for the Terraform Cloud Operator for Kubernetes to function properly, it must be able to make outbound requests over HTTPS (TCP port 443) to the Terraform Cloud application APIs. This may require perimeter networking as well as container host networking changes, depending on your environment. The IP ranges are documented in the [Terraform Cloud IP Ranges documentation](/terraform/cloud-docs/architectural-details/ip-ranges). The services which run on these IP ranges are described in the table below.
+- **Configurable synchronization**: The `--sync-period` option allows you to configure the synchronization frequency between custom resources and Terraform Cloud, ensuring timely updates and smoother operations.
 
-| Hostname         | Port/Protocol  | Directionality | Purpose                                                                                                            |
-| ---------------- | -------------- | -------------- | ------------------------------------------------------------------------------------------------------------------ |
-| app.terraform.io | tcp/443, HTTPS | Outbound       | Dynamically managing Terraform Cloud Workspaces and returning the output to Kubernetes via the Terraform Cloud API |
 
-### Compatibility
+## Supported Terraform Cloud features
 
-The current release of the Terraform Cloud Operator for Kubernetes supports the following versions:
+The Terraform Cloud Operator for Kubernetes allows you to create agent pools, deploy modules, and manage workspaces through Kubernetes controllers. These controllers enable you to automate and manage Terraform Cloud resources using custom resources in Kubernetes.
 
-* Helm 3.0.1 and above
-* Kubernetes 1.15 and above
+### Agent pools
 
-### Installation & Configuration
+Agent pools in Terraform Cloud manage the execution environment for Terraform runs. The Terraform Cloud Operator allows you to create and manage agent pools as part of your Kubernetes infrastructure.
```

Comment:
i'm reading the last sentence as the pools becoming part of the k8s infra - is that correct?

Reply:
Yup, that's correct! The agent get's spun up as a pod in the Kubernetes cluster and automatically connect to Terraform Cloud